src/tests/Common/XUnitWrapperGenerator/ITestInfo.cs

```diff
@@ -72,7 +80,7 @@ public override int GetHashCode()
         hash = hash * 23 + (TestNameExpression?.GetHashCode() ?? 0);
         hash = hash * 23 + (Method?.GetHashCode() ?? 0);
         hash = hash * 23 + (ContainingType?.GetHashCode() ?? 0);
-        hash = hash * 23 + (ExecutionStatement?.GetHashCode() ?? 0);
+        hash = hash * 23 + (_executionStatement?.GetHashCode() ?? 0);
```

Comment:
Can you use HashCode.Combine ?

Reply:
This code is in a source generator which targets .NET Standard 2.0.

Reply:
right, you'd have to have a package reference to the OOB. probably not worth it I guess.

Reply:
Referencing OOB dependencies for Roslyn components is very annoying to get right. I’d recommend against it unless necessary tbh.